src/app/fdctl/config/default.toml

```diff
@@ -906,6 +906,10 @@ dynamic_port_range = "8900-9000"
         # longer than this threshold.
         idle_timeout_millis = 2000
 
+        # Max delay for outgoing ACKs.
+        # A value of zero selects the hardcoded default ACK delay.
```

Comment:
Suggest not doing this. The default is the value in this file, there shouldn't be some other second level default if you set it to zero. Also would an operator ever even want to configure this?